warp10/src/main/java/io/warp10/script/functions/DELETE.java

```diff
@@ -52,19 +57,22 @@ public DELETE(String name) {
   
   @Override
   public Object apply(WarpScriptStack stack) throws WarpScriptException {
-    
     //
     // Extract expected number
     //
-    
+    boolean onlyDryRun = false;
+    long expected = Long.MAX_VALUE;
+
     Object o = stack.pop();
-    
-    if (!(o instanceof Long)) {
-      throw new WarpScriptException(getName() + " expects a count on top of the stack.");
+
+    if (o instanceof Long) {
+      expected = (long) o;
```

Comment:
Use ((Long) o).longValue(), autoboxing makes things harder to read and could lead to failing code.